internal/controller/apiextensions/composite/composition_functions.go

```diff
@@ -168,6 +195,14 @@ func WithComposedResourceObserver(g ComposedResourceObserver) FunctionComposerOp
 	}
 }
 
+// WithExtraResourcesGetter configures how the FunctionComposer should fetch extra
+// resources requested by functions.
+func WithExtraResourcesGetter(f ExtraResourcesGetter) FunctionComposerOption {
```

Comment:
Nit: Why `f`? I think it was probably short for `Fetcher`.

Reply:
yes, it was, it's now back to Fetcher though